packages/default-theme/components/SwProductTabs.vue

```diff
@@ -1,24 +1,103 @@
 <template>
   <div class="sw-product-tabs">
     <SfTabs class="product-details__tabs" :open-tab="openTab">
-      <slot />
+      <slot>
+        <SfTab title="Description">
+          <div>
+            <p>
+              {{ description }}
+            </p>
+          </div>
+        </SfTab>
+        <SfTab title="Properties">
+          <div class="product-details__properties">
+            <SfProperty
+              v-for="(property, i) in properties"
+              :key="i"
```

Comment:
it's always better to use unique id's from iterated elements, using an index is a tool only if we do not have such id
```suggestion
              v-for="property in properties"
              :key="property.id"
```